### PR TITLE
Add incidents permissions

### PR DESCRIPTION
**Pull request by BenAlfasi**

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -2461,6 +2461,41 @@ For an app to read or write all Windows update deployment settings with delegate
...
+| _Incidents.ReadWrite.All_ | Read and write to incidents | Allows the app to read and write incidents, on behalf of the signed-in user. | Yes | No |
+
+
+#### Application permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required |
+|:----------------|:------------------|:-------------|:-----------------------|
+| _Incidents.Read.All_ | Read all incidents | Allows the app to read all incidents, without a signed-in user. | Yes | No |
+| _Incidents.ReadWrite.All_ | Read and write to all incidents | Allows the app to read and write to all incidents, without a signed-in user. | Yes | No |
+
```
Comment (angelgolfer-ms) [inline concepts/permissions-reference.md:2480]: Lines 2478-2479:
Please delete the 5th column. The table header has 4 columns, consistent with the other tables for application permissions in the same topic.

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -2461,6 +2461,41 @@ For an app to read or write all Windows update deployment settings with delegate
...
+* _Incidents.Read.All_: Read incidents (`GET /security/incidents`)
+* _Incidents.ReadWrite.All_: Read incidents (`GET /security/incidents`)
+
+#### Application
+
+* _Incidents.Read.All_: Read all incidents (`GET /security/incidents`)
+* _Incidents.ReadWrite.All_: Read all incidents (`GET /security/incidents`)
+
+---
```
Comment (angelgolfer-ms) [inline concepts/permissions-reference.md:2497]: For each of the delegated usage (lines 2489 and 2490) and application usage (lines 2494 and 2495), can you please clarify the extent of incidents that an authorized app can actually read?

For instance, is the application usage about "Read all incidents in an organization"? And what about the delegated usage?

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
Hi @BenAlfasi , per your question in the related PR #13936 , yes if you can merge this PR into that one, that would be slightly better. If not, then this PR 14236 should be merged shortly before PR 13936, so that we don't get into a state that the new permissions in PR 13936 are lacking definition.

Please take a look at the couple of comments in this PR. Thanks!

**Comment by BenAlfasi** [comment]
Merged to #13936

File: concepts/permissions-reference.md (hunk 2 above)
Comment (angelgolfer-ms) [reply concepts/permissions-reference.md:2499]: @BenAlfasi , not sure of your answer to my comment/question above.
Would it be appropriate to qualify where you say "incidents..." in each of the lines 2472, 2473, 2480, 2481, 2491, 2492, 2496, 2497,  you say "incidents in an organization..."?

That is, for line 2491, you say:
"Read all incidents _in an organization_ that the user is allowed to read"

Where you currently say "Read all incidents..." or "Read or write all incidents...", for delegated and application permissions, we further quality the description to say:
"Read all incidents in an organization..." or "Read or write all incidents in an organization...",

Comment (BenAlfasi) [reply concepts/permissions-reference.md:2499]: @angelgolfer-ms Fixed in the Incidents API PR. #13936

**Pull request closed without merging**